static/app/components/contextPickerModal.tsx

```diff
@@ -69,6 +68,14 @@ type Props = ModalRenderProps & {
   allowAllProjectsSelection?: boolean;
 };
 
+function autoFocusReactSelect(reactSelectRef: any) {
+  if (!reactSelectRef) {
+    return;
+  }
+
+  reactSelectRef.select?.focus?.();
```

Comment:
nit: could just chain all the way

```suggestion
  reactSelectRef?.select?.focus?.();
```